keyboards/mtbkeys/mtb60/hotswap/readme.md

```diff
@@ -0,0 +1,29 @@
+# MTB60 (Hotswap Edition)
+
+## Details
+
+* Keyboard Maintainer: [MTBKeys](https://mtbkeys.com/)
+* Hardware Supported: *MTB60 PCB (Hotswap edition)*
+* Hardware Availability: [HB60 on MTBKeys Website](https://mtbkeys.com/)
+
+## Building Firmware
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Suggestion is now implemented